httpclient/4.5.3.wso2v1/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we use OSGi bundle? https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient-osgi/4.5.3